tools/ports/__init__.py

```diff
@@ -233,7 +237,7 @@ def erase():
 
   @staticmethod
   def get_build_dir():
-    return system_libs.get_build_dir()
+    return os.path.join(system_libs.get_build_dir(), 'ports')
```

Comment:
Is this needed?  I guess its nice to have system libs and ports cleanly separated maybe?

Reply:
No, it's not needed. My original version didn't have it but you suggested it in https://github.com/emscripten-core/emscripten/pull/23924#discussion_r1996094561 (I think I'd kind of prefer not having it).

Reply:
I don't feel strongly.  lgtm either way.